index.bs

```diff
@@ -270,6 +270,76 @@ one line except when a line break is definitely necessary.</p>
 
 </div>
 
+<h3 id=styling>Styling</h3>
+
+<p><i>This section is non-normative.</i></p>
+
+<p>CSS stylesheets that apply to an HTML page that contains a <a spec=html element>video</a> element
+can target WebVTT cues and regions in the video using the ''::cue'', ''::cue()'' and
+''::cue-region'' pseudo-elements.</p>
+
+<div class="example">
+
+ <p>In this example, an HTML page has a CSS stylesheet in a <a spec=html element>style</a> element
+ that styles all cues in the video with a gradient background and a text color, as well as changing
+ the text color for all <a>WebVTT Bold Objects</a> in cues in the video.</p>
+
+ <pre>
+ &lt;!doctype html>
+ &lt;html>
+  &lt;head>
+   &lt;title>Styling WebVTT cues&lt;/title>
+   &lt;style>
+    video::cue {
+      background-image: linear-gradient(to bottom, dimgray, lightgray);
+      color: papayawhip;
+    }
+    video::cue(b) {
+      color: peachpuff;
+    }
+   &lt;/style>
+  &lt;/head>
+  &lt;body>
+   &lt;video controls autoplay src="video.webm">
+    &lt;track default src="track.vtt">
+   &lt;/video>
+  &lt;/body>
+ &lt;/html>
+ </pre>
+
+</div>
+
+<p>CSS stylesheets can also be embedded in WebVTT files themselves.</p>
+
+<p>Style blocks are placed after any headers but before the first cue, and start with the line
+"STYLE". Blank lines and the string "<code>--></code>" cannot appear in the stylesheet. Comment
```

Comment:
It doesn't discard the block, it just ends the block and starts a new cue. But yeah, https://github.com/w3c/webvtt/commit/adcdbf12e20413e99e908aa209b2e40030c03e78